src/link/link_info/info_data.rs

```diff
@@ -42,6 +42,7 @@ pub enum InfoData {
     MacSec(Vec<InfoMacSec>),
     Hsr(Vec<InfoHsr>),
     Geneve(Vec<InfoGeneve>),
+    IpTun(Vec<InfoIpTun>),
```

Comment:
The `IpTun` might confuse with `Tun` which sounds like a user space interface. Could we use `IpTunnel`?

Reply:
Sure

Reply:
Done, thank you!